Main Server/components/hlsPlayer.js

```diff
@@ -921,7 +966,7 @@ export default class HlsPlayer extends Component {
         this.hideControlsTimeout = setTimeout(() => {
             this.setState({ controlsVisible: false });
             this.videoContainer.style.cursor = 'none';
-        }, 5000);
+        }, 5000000000000000000);

```

Comment:
?

Reply:
Increased time so that the controls would be visible for longer time during testing this is now reverte back to 5 sec again.